kong/plugins/acl/handler.lua

```diff
@@ -60,14 +89,17 @@ function ACLHandler:access(conf)
   if next(conf.whitelist or empty) then
     if not next(acls or empty) then
       block = true
+
```

Comment:
ditto: all those style changes - while appreciated, it'd be great if they could at least be included in a separate commit :) Thanks!